pages/_app.js

```diff
@@ -0,0 +1,27 @@
+import App, { Container } from 'next/app'
+import Head from 'next/head'
+
+export default class OscoinApp extends App {
+  // static async getInitialProps({ Component, ctx }) {
+  //   let pageProps = {}
+
+  //   if (Component.getInitialProps) {
+  //     pageProps = await Component.getInitialProps(ctx)
+  //   }
+
+  //   return { pageProps }
+  // }
```

Comment:
Is this commented code supposed to go into the repo?

Reply:
No, good catch! I removed it to give pages the ability to be statically pre-rendered. Nextjs determiens if something needs to be dynamically created on the fly by checking the presence of `getInitialProps` - if this is present in `_app` (which is shared by all pages) it will always infer dynamic creation.

Reply:
Got it. Let's remove it then.